voxblox/include/voxblox/integrator/merge_integrator.h

```diff
@@ -19,7 +19,7 @@ class MergeIntegrator {
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
 
   // all methods are static so the class does not need a constructor
-  MergeIntegrator() = delete;
+  // MergeIntegrator() = delete;
```

Comment:
Is there a reason the constructor is required here?
In general I think this class should probably be a set of free functions, because there is no state and just a single static const member variable that doesn't really justify the existence of this class, a namespace would do. Would this be a problem for you if we did this in the future? @genemerewether (@helenol @ZacharyTaylor wdyt? [link](https://stackoverflow.com/questions/14698924/for-a-static-class-with-private-constructor-in-c-should-i-define-a-destruct))

Reply:
Agree with @mfehr on maybe namespacing making more sense with the current implementation of the merge integrator.

Reply:
Yeah I think I just had them in a class to group the functions together, but your right a namespace works and is probably clearer.

Reply:
No problem for me if it becomes a namespace later :)